Utilities/Logging.cs

```diff
@@ -298,124 +320,13 @@ public static void ExceptionHandler(Exception exception)
                 { "StackTrace", exception.StackTrace ?? "StackTrace not available" }
             };
 
-            if (isUniqueMessage(exception.GetType() + ": " + exception.Message, trace))
-            {
-                Logging.Info("Reporting unhandled exception, anonymous ID " + RollbarLocator.RollbarInstance.Config.Person.Id + ":" + exception);
-                RollbarLocator.RollbarInstance.Error(exception, trace);
-            }
+            Logging.Info("Reporting unhandled exception, anonymous ID " + RollbarLocator.RollbarInstance.Config.Person.Id + ":" + exception);
```

Comment:
Did we intend to lose the uniqueness test here?

Reply:
We were only testing for uniqueness on info level messages and we were only sending error level messages. Consequently, we were already not testing any messages for uniqueness.